chef/mixlib-install
# CLI UX and Testing improvements

CLI improvements include:

* Use `enum` option where appropriate (prints valid options to users)
* Set missing defaults
* Add basic architecture option validation

Testing improvements include:

* Collapse unit and functional specs into single `specs/*` directory
* Use RVM and multiple Ruby versions to drive all testing on Travis
* Update naming of Rake tasks to match idioms used in other projects like [chef/omnibus](https://github.com/chef/omnibus).
* Fix Windows package naming for Inspec package check in `powershell_install_ps1` acceptance suite

/cc @chef/engineering-services
